Dockerfile

```diff
@@ -0,0 +1,18 @@
+ARG NEXUS_VERSION=3.19.0-SNAPSHOT
+
+FROM maven:3-jdk-8-alpine AS build
+ARG NEXUS_VERSION=3.19.0
+ARG NEXUS_BUILD=SNAPSHOT
+
+COPY . /nexus-repository-apk/
+RUN cd /nexus-repository-apk/; sed -i "s/3.19.0-02/${NEXUS_VERSION}-${NEXUS_BUILD}/g" pom.xml; \
+    mvn clean package -PbuildKar;
+
+FROM sonatype/nexus3:$NEXUS_VERSION
+ARG NEXUS_VERSION=3.19.0
+ARG NEXUS_BUILD=SNAPSHOT
```

Comment:
This looks unused? Should it be removed?

Reply:
I managed to get rid of most of the NXRM version specific stuff in the Dockerfile. Here is a simpler [Dockerfile](https://github.com/sonatype-nexus-community/nexus-format-archetype/blob/master/src/test/resources/projects/it1/reference/Dockerfile) from the nexus-format-archetype code base. You could just replace all occurrences of "foo" with "apk", and change the FORMAT_VERSION ARG value from "0.0.1" to "0.0.1-SNAPSHOT".

Reply:
In: 72661e1